code/datums/craft/recipes/misc.dm

```diff
@@ -3,7 +3,7 @@
 	result = /obj/structure/girder
 	time = WORKTIME_SLOW
 	steps = list(
-		list(CRAFT_MATERIAL, MATERIAL_STEEL, 5)
+		list(CRAFT_MATERIAL, MATERIAL_STEEL, 25)
```

Comment:
this is a pretty crazy change, these are needed to build walls.
400% increases in game balance values need some explanation